### PR TITLE
doc: focus on v3 in README

### LINKED ISSUE #5192: rewrite readme for etcd3 release
The current readme is v2 api centric. We want to rewrite it to make it etcd3 centric. Also we will link the doc back to a copied version of v2 doc.

### PR DESCRIPTION
**Pull request by heyitsanthony**
Fixes #5192

### CODE REVIEW

**Comment by gyuho** [comment]
lgtm. defer to @xiang90

File: Documentation/README_v2.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+# etcd
```
Comment (xiang90) [inline Documentation/README_v2.md]: probably we should just delete this. we will copy the entire v2 doc from 2.3 release to here.

**Comment by xiang90** [comment]
LGTM

File: Documentation/README_v2.md (hunk 1 above)
Comment (heyitsanthony) [inline Documentation/README_v2.md]: so should I get rid of the link at the top of README.md pointing to the old v2 README too?

Comment (xiang90) [inline Documentation/README_v2.md]: keep it a todo?

Comment (heyitsanthony) [inline Documentation/README_v2.md]: I'll just move it now I guess...

Comment (xiang90) [inline Documentation/README_v2.md]: sure.

**Comment by heyitsanthony** [comment]
@xiang90 cloned the docs to `Documentation/v2`. Kept the README but titled it `etcd2` so there's some sort of landing page for v2 instead of a directory listing of md files. ptal

**Comment by xiang90** [comment]
OK. Let's clean other stuff up in follow up prs. LGTM

**Pull request merged**